### PR TITLE
WIP: Login Support for Fabrik

### PR DESCRIPTION
**Pull request by Ram81**
@deshraj @kdexd @ayshrv @utsavgarg this PR adds support for user login in Fabrik using social apps using django allauth.

### CODE REVIEW

**Comment by pavanagrawal123** [comment]
Oh I'm glad the previous code came into use! I remember when I had authored this PR, I had accidentally left a debug page in there in the templates folder, and I see it's still there. If you don't think we need it, maybe it should be removed.

**Comment by Ram81** [comment]
Yeah

File: backendAPI/templates/login.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- This is purely for debugging purposes. This will not be served to frontend users. -->
```
Comment (utsavgarg) [inline backendAPI/templates/login.html]: Do we need this file?

File: backendAPI/tests.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (utsavgarg) [inline backendAPI/tests.py]: Do we need this?

File: backendAPI/templates/login.html (hunk 1 above)
Comment (Ram81) [reply backendAPI/templates/login.html]: No, I''ll remove it

File: backendAPI/tests.py (hunk 2 above)
Comment (Ram81) [reply backendAPI/tests.py]: No, I'll remove this

**Comment by utsavgarg** [comment]
@Ram81 LGTM :) Merging this for now so you are not blocked, but please remove the contents of `backendAPI/models.py` in a future PR.

**Pull request merged**